compiler/onecc-docker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:18.04 
+
+RUN apt update -y && \ 
+	 apt install -y \ 
+	 wget && \ 
+	 apt clean 
+RUN wget https://github.com/Samsung/ONE/releases/download/1.20.0/one-compiler_1.20.0_amd64.deb && \ 
+	 apt-get install -y ./one-compiler_1.20.0_amd64.deb 
+ENTRYPOINT ["onecc"]
```

Comment:
please add new line here.
![image](https://user-images.githubusercontent.com/4616940/190549030-a85edc8a-1fc0-481e-8b25-03bfa61a77dd.png)
please check before posting PRs that you should not see these red circle.